src/parser/test/hardcoded_tests.js

```diff
@@ -2590,5 +2590,22 @@ module.exports = {
         }
       }
     }
+  },
+  'Valid export/import reserved words': {
```

Comment:
Wow this is complicated! Took me a few minutes to grok what was and wasn't allowed and why!